src/main/java/net/dv8tion/jda/core/JDA.java

```diff
@@ -743,34 +735,42 @@ public boolean equals(Object o)
 
     /**
      * Shuts down JDA, closing all its connections.
-     *
-     * <p>This is the same as calling {@link #shutdown(boolean) shutdown(true)}.
+     * After this command is issued the JDA Instance can not be used anymore.
+     * Already enqueued {@link net.dv8tion.jda.core.requests.RestAction RestActions} are still be executed.
      */
     void shutdown();
```

Comment:
A long as any non-deamon thread still keeps the JVM alive yes. `Requester#request()` is called when queueing RestActions, that's why only ***already enqueued** RestActions are still executed.*

The only problem is that JDA doesn't have any non-deamon threds at that time which could be fixed by making the executor(s) non-deamon.